### PR TITLE
Arm64v8

### PR DESCRIPTION
**Pull request by dalir**

### CODE REVIEW

File: alpine-arm64v8/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+COPY jenkins-slave /usr/local/bin/jenkins-slave
+
+# Root user is required to interact with Docker via docker.sock
+USER root
+
+RUN echo 'http://dl-cdn.alpinelinux.org/alpine/v3.7/community' >> /etc/apk/repositories \
+  && apk --update add make \
+  bash \
+  docker \
```
Comment (blaines) [inline alpine-arm64v8/Dockerfile]: Can we confirm the version here? 17.06.0-ce or better

File: alpine-arm64v8/Dockerfile.build [hunk 2]
```diff
@@ -0,0 +1,50 @@
+FROM alpine:3.6
```
Comment (blaines) [inline alpine-arm64v8/Dockerfile.build]: rm this file

**Review by blaines — CHANGES_REQUESTED** [review]
We should remove the .build files, also add a Makefile for automated build of the images.

File: alpine-arm64v8/Dockerfile (hunk 1 above)
Comment (dalir) [reply alpine-arm64v8/Dockerfile]: 17.10.0

Comment (dalir) [reply alpine-arm64v8/Dockerfile]: Thought it's not a bad idea to have always the latest stable in the build

File: alpine-arm64v8/Dockerfile.build (hunk 2 above)
Comment (dalir) [reply alpine-arm64v8/Dockerfile.build]: will do.

**Review by blaines — APPROVED** [review]

File: alpine-arm64v8/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+
+VOLUME /home/jenkins/.jenkins
+VOLUME ${AGENT_WORKDIR}
+WORKDIR /home/jenkins
+
+COPY jenkins-slave /usr/local/bin/jenkins-slave
+
+# Root user is required to interact with Docker via docker.sock
+USER root
+ARG DOCKER_VERSION 17.10.0-r0
```
Comment (blaines) [inline alpine-arm64v8/Dockerfile]: I believe you forgot an =

File: alpine/Dockerfile [hunk 4]
```diff
@@ -23,56 +23,15 @@ FROM jenkinsci/jnlp-slave:alpine
...
 USER root
-# BEGIN from Docker https://github.com/docker-library/docker/blob/master/17.06/Dockerfile
-RUN apk add --no-cache \
-		ca-certificates
-ENV DOCKER_CHANNEL stable
-ENV DOCKER_VERSION 17.09.0-ce
-ENV DOCKER_COMPOSE_VERSION 1.16.1
+
 ENV SAM_VERSION 0.2.4
+ARG DOCKER_VERSION 17.10.0-r0
```
Comment (blaines) [inline alpine/Dockerfile]: Same as other comment "="

**Review by blaines — CHANGES_REQUESTED** [review]

**Pull request merged**